examples/rpc_publish.sh

```diff
@@ -29,9 +31,10 @@ CONFIG="/tmp/config.toml"
 P2P_PUBLIC_KEY="/tmp/polka-storage/public.pem"
 P2P_PRIVATE_KEY="/tmp/polka-storage/private.pem"
 P2P_ADDRESS="/ip4/127.0.0.1/tcp/62649"
+# Deal parameters JSON location
+DEAL_PARAMS="/tmp/deal_params.json"
```

Comment:
We should scope everything to `/tmp/polka-storage`.
btw, instead of `/tmp` we should use `$TMPDIR` (a recent issue I struck upon on our GPU runner).